packages/@n8n/ai-workflow-builder.ee/src/utils/stream-processor.ts

```diff
@@ -117,19 +118,30 @@ export function processStreamChunk(streamMode: string, chunk: unknown): StreamOu
 		}
 
 		// Handle plan creation
-		if (agentChunk?.createPlan?.workflowPlan) {
-			const workflowPlan = agentChunk.createPlan.workflowPlan as WorkflowPlan;
+		if (agentChunk?.create_plan?.workflowPlan) {
+			const workflowPlan = agentChunk.create_plan.workflowPlan as WorkflowPlan;
 			const planChunk = {
 				role: 'assistant' as const,
 				type: 'plan' as const,
 				plan: workflowPlan.plan,
 				message: workflowPlan.intro,
 			};
 			return { messages: [planChunk] };
+		} else if ((agentChunk?.create_plan?.messages ?? []).length > 0) {
+			// When planner didn't create a plan, but responded with a message
+			const lastMessage =
+				agentChunk.create_plan!.messages![agentChunk.create_plan!.messages!.length - 1];
+			const messageChunk: AgentMessageChunk = {
+				role: 'assistant',
+				type: 'message',
+				text: lastMessage.content as string,
+			};
+
+			return { messages: [messageChunk] };
 		}
 
-		if (agentChunk?.adjustPlan?.workflowPlan) {
-			const workflowPlan = agentChunk.adjustPlan.workflowPlan as WorkflowPlan;
+		if (agentChunk?.adjust_plan?.workflowPlan) {
+			const workflowPlan = agentChunk.adjust_plan.workflowPlan as WorkflowPlan;
```

Comment:
Would be good to avoid type casting where possible. Type guards here would be better. 

But I know this stream function types needs to be refactored, like we discussed on the other PR.